integration-tests/tzip16-originate-contracts-with-metadata-on-HTTPS-and-sha256-hash-and-fetch-metadata.spec.ts

```diff
@@ -11,14 +11,14 @@ CONFIGS().forEach(({ lib, rpc, setup }) => {
 
     const test = require('jest-retries');
```

Comment:
should be removed?

Reply:
Removed that line and the retry calls in each test.